### DISCUSSION TITLE
pages setting

### DISCUSSION THREAD

**Discussion by 1986-OldMan** [community/community#45958] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello to everyone.
i'm first time here to ask something and i need help with a problem in pages setting for my repositories.

To certain repositories work to see my project.,but from certain repositories not work to see my project.

In pages settings at build and deployment i select "deploy from a branch" and at branch i select "master" as main branch and select folder "(/root)",and after i'm going to my repositoies and go for environments to see my project on github and instead to see my project ,i see my readme.md and title of repository.

![problem](https://user-images.githubusercontent.com/114245975/215897308-6cd1bd61-75cb-4a8e-88e6-491d99cca18d.jpg)

Any idea for this problem.?hank you.

**Comment by jge162** [comment, marked as answer]
Double check that your is correctly formatted -> `index.html` file in the root directory of your repository.

Verify your repository name is a valid domain name and can be accessed on the web using
the format ->`https://<username>.github.io/<repository-name>/.`

**Reply by 1986-OldMan** [reply]
Thanks for help, it work now.
